.github/workflows/build-linux.yml

```diff
@@ -0,0 +1,79 @@
+name: Build and Package - Linux
+
+on: [push, pull_request]
+
+jobs:
+  build-hnsd:
+    runs-on: ubuntu-18.04
+
+    steps:
+      - name: Checkout hnsd repository
+        uses: actions/checkout@v2
+        with:
+          repository: 'handshake-org/hnsd'
+
+      - name: Install dependencies
+        run: sudo apt install -y unbound libunbound-dev
```

Comment:
Is there a reason for installing both `unbound` and `libunbound-dev` here?

Reply:
Removed unbound.